src/lib/tabs/tab-body.html

```diff
@@ -1,4 +1,6 @@
 <div class="mat-tab-body-content" #content
+     (swipeleft)="onSwipe.emit($event)"
+     (swiperight)="onSwipe.emit($event)"
```

Comment:
I only want to listen to left/right events. 

Also by listening to each event separately, I get distinct event types in the event handler vs just `swipe`.